source/lib/orchestrator-log-stack.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as cdk from 'aws-cdk-lib';
+import { LogGroup, CfnLogGroup, RetentionDays } from 'aws-cdk-lib/aws-logs';
+import { Key } from 'aws-cdk-lib/aws-kms';
+
+export interface OrchLogStackProps extends cdk.StackProps {
+  logGroupName: string;
+  solutionId: string;
+}
+
+export class OrchLogStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props: OrchLogStackProps) {
+    super(scope, id, props);
+    const stack = cdk.Stack.of(this);
+
+    const reuseOrchLogGroup = new cdk.CfnParameter(this, 'Reuse Log Group', {
+      type: 'String',
+      description: `Reuse existing Orchestrator Log Group? Choose "yes" if the log group already exists, else "no"`,
+      default: 'no',
+      allowedValues: ['yes', 'no'],
+    });
+    reuseOrchLogGroup.overrideLogicalId(`ReuseOrchestratorLogGroup`);
+
+    const kmsKeyArn = new cdk.CfnParameter(this, 'KMS Key Arn', {
+      type: 'String',
+      description: `ARN of the KMS key to use to encrypt log data.`,
+    });
+    kmsKeyArn.overrideLogicalId(`KmsKeyArn`);
+
+    /**********************
+     * Encrypted log group
+     */
+    // As of March 2021, CWLogs encryption is not yet supported in GovCloud
```

Comment:
Thanks for comments like this, explaining intent. Very useful.

Reply:
I think this is no longer true.